fix compilation error "no match for operator+=" when building with USE_SRA=1 because of type mismatch in buggy code
see also https://github.com/DaehwanKimLab/centrifuge/issues/191
author: Kenneth Hoste (HPC-UGent), with help from Lars Viklund (Umea University)
--- centrifuge-1.0.4-beta/centrifuge.cpp.orig	2020-06-08 20:33:19.578955606 +0200
+++ centrifuge-1.0.4-beta/centrifuge.cpp	2020-06-08 20:33:26.219065266 +0200
@@ -3006,7 +3006,7 @@
 
 	int fileCnt = mates1.size() + queries.size(); // the order should be consistent with the wrapper
 #ifdef USE_SRA
-	fileCnt += sra_accs ;
+	fileCnt += sra_accs.size() ;
 #endif
 
 	for ( int fileIdx = 0 ; fileIdx < fileCnt ; ++fileIdx )
